.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: php
+
+php:
+    - 5.4
+    - 5.5
+    - 5.6
+    - hhvm
+
+matrix:
+    allow_failures:
+        - php: hhvm
+
+before_script:
+  - COMPOSER_ROOT_VERSION=dev-master composer install --dev --prefer-source
```

Comment:
Sorry, the COMPOSER_ROOT_VERSION doesn't seem relevant, because there's no circular reference.